Packs/ApiModules/Scripts/MicrosoftApiModule/MicrosoftApiModule.py

```diff
@@ -185,7 +190,7 @@ def http_request(
         response = super()._http_request(  # type: ignore[misc]
             *args, resp_type="response", headers=default_headers, **kwargs)
 
-        if should_http_retry_on_rate_limit:
+        if should_http_retry_on_rate_limit and self.is_command_executed_from_integration():
```

Comment:
@amshamah419 @YuvHayun I think we should open a devbug to add this in csp in return_results, but we shouldnt do that fix in the context of a bug

Reply:
I already updated the PMs that we should find a broader solution.